wagtailimporter/serializer.py

```diff
@@ -226,39 +236,33 @@ def db_filename(self):
         """
 
         folder_name = 'original_images'
-        filename = ('images/%s' % self.file).replace('/', '')
+        filename = 'images/%s' % self.file.replace('/', '-')
         filename = "".join((i if ord(i) < 128 else '_')
                            for i in unidecode(filename))
 
-        while len(os.path.join(folder_name, filename)) >= 95:
-            prefix, dot, extension = filename.rpartition('.')
-            filename = prefix[:-1] + dot + extension
+        max_length = 95
```

Comment:
Hmm, random magic number, what do you mean past Danni.

Reply:
At a guess, this logic seems to be a copy of Wagtails [AbstractImage.get_upload_to()](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailimages/models.py#L114-L131) which has the magic 95 number as well, although it also has a link to a Django ticket with more information.